.github/workflows/feature-list.yml

```
@@ -40,74 +40,48 @@ jobs:
           const csv = require('csvtojson');
 
           const headers = [
-            'Theme (also: Keychain Name)',
+            'Theme',
             'Category',
             'Function',
             'Feature',
-            'Primary Persona',
-            'Entitlement',
             'Subscription Tier',
-            'Free Comparison Tier',
-            'TeamDesigner Comparison Tier',
-            'TeamOperator Comparison Tier',
-            'Enterprise Comparison Tier',
-            'Tech',
-            'Version',
-            'Engineer',
+            'Free Tier',
+            'TeamDesigner Tier',
+            'TeamOperator Tier',
+            'Enterprise Tier',
             'Pricing Page?',
             'Video?',
             'Documented?',
-            'Anonymous',
-            'Authorization',
-            'Team Admin',
-            'Workspace Admin',
-            'Org Billing Manager',
-            'Org Admin',
-            'Provider Admin',
-            'Curator',
-            'MeshMap',
-            'Keychain ID',
-            'SQL',
-            'Key ID',
-            'Inserted',
-            'Local Provider',
-            'Update SQL',
-            'E2E Test',
           ];
 
           csv({
-            noheader: true, // Ignore the first row as headers
-            headers: headers, // Use our custom headers
-            output: 'json', // Output as JSON
+            noheader: true, 
+            headers: headers,
+            output: 'json', 
           })
             .fromFile('spreadsheet.csv')
             .then(rows => {
 
               // Filter data
               const filteredData = rows
                 .map(row => {
-                  const pricingPage = row['Pricing Page?']?.toLowerCase() || '';
-                  const documented = row['Documented?']?.trim() || '';
-
-                  const includeRow =
-                    (pricingPage && ['x', 'X', 'true'].includes(pricingPage)) ||
-                    (documented.startsWith('https://docs.meshery.io/') ||
-                      documented.startsWith('https://docs.layer5.io/'));
-                  if (!includeRow) return null;
-
-                  return {
-                    theme: row['Theme (also: Keychain Name)'],
+                 const pricingPage = row['Pricing Page?']?.toLowerCase() || '';
+                 const hasXTier = ['Free Tier', 'TeamDesigner Tier', 'TeamOperator Tier', 'Enterprise Tier'].some(tier => row[tier]?.trim().toLowerCase() === 'x');
+                 const includRow = hasXTier || (pricingPage && ['x', 'X'].includes(pricingPage));
```

```suggestion
                 const includeRow = hasXTier || (pricingPage && ['x', 'X'].includes(pricingPage));
```

Missing an "e" here.